store/sqlstore/post_store.go

```diff
@@ -2411,12 +2464,10 @@ func (s *SqlPostStore) cleanupThreads(postId, rootId string, permanent bool, use
 			if s.DriverName() == model.DatabaseDriverPostgres {
```

Comment:
Wanted to comment in L2461, but GitHub won't let me.

It would be good to add a comment in the lines of "Updating replyCount and reducing participants if this was the last post in the thread for the user" to explain what is going on here.